charts/latest/azurefile-csi-driver/templates/rbac-csi-azurefile-node.yaml

```diff
@@ -4,27 +4,24 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   namespace: {{ .Release.Namespace }}
-  name: csi-azurefile-secret-role
+  name: csi-azurefile-node-secret-role
 rules:
   - apiGroups: [""]
     resources: ["secrets"]
-    verbs: ["get", "list", "create"]
```

Comment:
for azure file driver, need `create` permission, right?

Reply:
just controller needs `create` permission.